test/integration/graphql_test.lua

```diff
@@ -1639,3 +1639,39 @@ function g.test_schema_input_arg_described_with_kind_variable_pass()
     local _, errors = check_request(query, query_schema, nil, nil, { variables = variables })
     t.assert_equals(errors, nil)
 end
+
+function g.test_arguments_default_values()
+    local function callback(_, _)
+        return nil
+    end
```

Comment:
It would be good to check that the default value is accessible here.

Reply:
This is not the subject of this pull request.